src/db.rs

```diff
@@ -607,3 +666,86 @@ impl Database {
         GridFsBucket::new(self.clone(), options.into().unwrap_or_default())
     }
 }
+
+pub(super) struct RunCursorCommand<'conn> {
```

Comment:
We have each of our operation structs in their own file under `src/operation` -- can we move this one there as well as `src/operation/run_cursor_command.rs`?